contracts/lootbox-v1/hardhat.config.js

```diff
@@ -0,0 +1,21 @@
+require("@nomiclabs/hardhat-waffle");
+
+// This is a sample Hardhat task. To learn how to create your own go to
+// https://hardhat.org/guides/create-task.html
+task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
+  const accounts = await hre.ethers.getSigners();
+
+  for (const account of accounts) {
+    console.log(account.address);
+  }
+});
+
+// You need to export an object to set up your config
+// Go to https://hardhat.org/config/ to learn more
+
+/**
+ * @type import('hardhat/config').HardhatUserConfig
+ */
+module.exports = {
+  solidity: "0.8.4",
```

Comment:
I am 100% sure that the compiling contracts won't work

Reply:
The solidity compiler version is different from pragma version in contract.

Reply:
The solidity compiler version is different from pragma version in contract.

Reply:
> I am 100% sure that the compiling contracts won't work

KEK

Reply:
Regarding about the compile the smart contract I did test it before implementing the WyvernExchange and it worked fine.
The main version differences are the WyvernExchange and compiler setting but it could be fixed.
But for now we gonna use Genie Aggregator instead of Wyvern Exchange interface and I am sure it will work fine with Genie

Reply:
'Compiling wont work' means that 'npx hardhat compile' won't work. :) The hardhat compiler version should be specified properly. If not, hardhat won't be able to compile.